content/_index.md

```diff
@@ -36,6 +36,12 @@ for enrollment details!
     alt="Nia Bailey standing in front of the Public Library in New York"
     caption="Nia Bailey completed the UI/UX Designer track in 2021." >}}
 
+{{% if-equal enroll false %}}
+# Do you share our mission to help underrepresented minorities get access to training so that they can build careers in tech?
+
+<a class="button-like standout" href="/partner" target="_blank">Partner with Us</a>
```

Comment:
Please remove target="_blank".  I don't think we want to open a new window in this case.